README.md

```diff
@@ -33,19 +33,19 @@ A sample app for testing purposes is automatically built after commits in the re
 ### Screenshots of a sample Android app
 
 <p align="center">
-<img src="docs/images/app-drawer.png" height = "480" width="270"> <img src="docs/images/app-tracks.png" height = "480" width="270"> <img src="docs/images/app-schedule.png" height = "480" width="270">
+<img src="docs/_static/images/app-drawer.png" height = "480" width="270"> <img src="docs/_static/images/app-tracks.png" height = "480" width="270"> <img src="docs/_static/images/app-schedule.png" height = "480" width="270">
```

Comment:
@pri22296 because of big image size not rendering properly. now I have changes from three images per row to two images per row. please check http://yaydoc.herokuapp.com/preview/rbalajis25@gmail.com/14104444-d7b7-4d5d-bddd-eecb76b8f500_preview/